packages/cli-kit/src/git.ts

```diff
@@ -60,6 +110,42 @@ export async function downloadRepository({
   }
 }
 
+export async function getLatestCommit(directory?: string) {
+  try {
+    const logs = await git({baseDir: directory}).log({
+      maxCount: 1,
+    })
+    if (!logs.latest) throw NoCommitError()
+    return logs.latest
+  } catch {
+    throw NoCommitError()
+  }
+}
+
+export async function commitAll(message: string, options?: {directory?: string; author?: string}) {
+  const simpleGit = git({baseDir: options?.directory})
+  const status = await simpleGit.status()
+
+  if (!status.files.length) throw NothingToCommitError()
+
+  await simpleGit.add(status.files.map((file) => file.path))
```

Comment:
Given there is a difference, I think `commitAll` is a misnomer, as I'd expect it to do the equivalent of `git commit --all`.

Instead, maybe make 2 functions:

* `addAll` which runs `git add --all`
* `commit` which runs - you guessed it! - `git commit`

As an aside, I think that would leave each function with a clear single responsibility, and it would also save us from running in Node logic that should really just be a flag to a git command.